manifests/config.pp

```diff
@@ -8,7 +8,11 @@
     Class['puppet::server::config'] ~> Class['foreman_proxy::service']
   }
 
-  if $foreman_proxy::tftp     { include ::foreman_proxy::tftp }
+  if $foreman_proxy::tftp {
+    class { '::foreman_proxy::tftp':
+      tftp_managed => $foreman_proxy::tftp_managed,
```

Comment:
I don't think this needs to be set/duplicated here, the default's already set correctly.